app/views/dashboard/new_landing_page.html.slim

```diff
@@ -1,86 +1,83 @@
--unless params[:search]
+- unless params[:search]
   .carousel
```

Comment:
Is it preferred style to add a space between `-` or `=` in slim templates?  We've generally connected those operators to the code that follows them in the past.

Unless there's a standard that we've messed up on--which is entirely possible--we'd prefer to keep code consistent with the rest of the project.